src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -773,5 +772,7 @@ getEndingTxId bh = callDb "getEndingTxId" $ \db -> do
     convertInt (SInt thing) = fromIntegral thing
     convertInt _ = error "impossible"
 
+-- Careful doing this! It's expensive and for our use case, probably pointless.
+-- We should reserve vacuuming for an offline process
```

Comment:
Maybe a command line switch, so that users know what they can expect and aren't surprised when startup takes a long time?

Reply:
Actually, should we just provide a separate tool (or mode of Chainweb-node) that only does off-line db maintenance. Like consistency checks, repairs, pruning, GC, etc?

Reply:
We'll just delete it for now